### PR TITLE
docs: remove tox from contributing docs

### PR DESCRIPTION
**Pull request by steven-murray**
This removes references to `tox` from the contributing docs, since they're outdated.

I'd be keen to hear from @andreimesinger if this updated instructions work well.

### CODE REVIEW

**Comment by BradGreig** [comment]
Hey @steven-murray, realistically I will not realistically have the time to look at this for at least 3 weeks. Really sorry, just got a bunch of stuff coming up (plus a week off). Hopefully @qyx268 will have the time?

**Comment by steven-murray** [comment]
> Hey @steven-murray, realistically I will not realistically have the time to look at this for at least 3 weeks. Really sorry, just got a bunch of stuff coming up (plus a week off). Hopefully @qyx268 will have the time?

No problem, thanks for letting me know, @BradGreig !

Thanks @qyx268 .

**Comment by qyx268** [comment]
this one should be closed

File: .github/workflows/test_suite.yaml [hunk 1]
```diff
@@ -20,86 +20,85 @@ jobs:
       ENV_NAME: tests
       PYTHON: ${{ matrix.python-version }}
       OS: ${{ matrix.os }}
+      CC: gcc
```
Comment (andreimesinger) [inline .github/workflows/test_suite.yaml:23]: so for me, this wouldn't have worked... i couldn't get the OS to allow me to alias gcc to the non-clang version.  It is seems hardcoded to point to /usr/bin/gcc.  So I had to install by pre-pending the homebrew compiler version, i.e. gcc-11 that is in /usr/local/bin:

$ which gcc
/usr/bin/gcc
$ which gcc-11
/usr/local/bin/gcc-11

$ gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/4.2.1
Apple clang version 13.0.0 (clang-1300.0.29.3)
Target: x86_64-apple-darwin20.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

$ gcc-11 --version
gcc-11 (Homebrew GCC 11.2.0) 11.2.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

**Review by andreimesinger — CHANGES_REQUESTED** [review]
as i noted, for me i had to use CC=gcc-11.  CC=gcc doesn't work.

also i recommend adding an example when you say "prepend" the installation pip install with the LIB, INC, CFLAGS paths, because I didn't know the order of the paths.  so just saying something like, "e.g. $ CC=gcc-11 CFLAGS="-isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX11.3.sdk" pip install -e .[dev] "

**Comment by steven-murray** [comment]
Thanks @andreimesinger, I'll add that. Why MacOS has to be be so difficult, I don't know.

**Review by andreimesinger — APPROVED** [review]

**Pull request merged**